src/osd/PrimaryLogPG.cc

```diff
@@ -755,7 +761,7 @@ void PrimaryLogPG::maybe_force_recovery()
        ++it) {
     if (*it == get_primary()) continue;
     pg_shard_t peer = *it;
-    if (peer_missing.count(peer) &&
+      if (peer_missing.find(peer) != peer_missing.end() &&
```

Comment:
Like here:
<pre>
auto it = peer_missing.find(peer);
if (it != peer_missing.end() &&
   !it->second.get_rmissing.empty()) {
...
</pre>